platform-test/src/main/java/org/killbill/billing/platform/test/PlatformDBTestingHelper.java

```diff
@@ -57,12 +58,22 @@ protected PlatformDBTestingHelper() {
         if ("true".equals(System.getProperty("org.killbill.billing.dbi.test.h2"))) {
             log.info("Using h2 as the embedded database");
             instance = new H2EmbeddedDB();
+        } else if ("true".equals(System.getProperty("org.killbill.billing.dbi.test.postgresql"))) {
+            if (isUsingLocalInstance()) {
+                log.info("Using postgresql local database");
+                final String databaseName = System.getProperty("org.killbill.billing.dbi.test.localDb.database", "postgres");
+                final String username = System.getProperty("org.killbill.billing.dbi.test.localDb.username", "postgres");
+                final String password = System.getProperty("org.killbill.billing.dbi.test.localDb.password", "postgres");
+                instance = new PostgreSQLStandaloneDB(databaseName, username, password);
+            } else {
+                throw new UnsupportedOperationException("PostgreSQL can be chosen for stand-alone mode; set org.killbill.billing.dbi.test.useLocalDb to true.");
```

Comment:
See https://github.com/killbill/killbill-commons/commit/bbeb3af036345d630d5f29501697ed8da9ceac37.

Do you know what is the effort to make the airlift embedded server work on Windows? Probably the [tar](https://github.com/airlift/testing-postgresql-server/blob/master/src/main/java/io/airlift/testing/postgresql/EmbeddedPostgreSql.java#L288) invocation should change, anything else?